coralproject/talk
# [CORL-758] Show new comments in stream when sorting by oldest

## What does this PR do?

Push approved/new comments to the end of the stream when sorting by oldest.

## How do I test this PR?

1. Go to an non-pre-modded stream
2. Select Sort By: Oldest
3. Create a new comment that won't get flagged for toxicity
4. See the comment at bottom of stream above the "Load More" button
